File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/ct5/bucketsort.java
--- 
+++ 
@@ -6,7 +6,7 @@
 
     ArrayList<Integer> sorted_arr = new ArrayList<Integer>(100);
     int i = 0;
-    for (Integer count : arr) { // arr is counts in fixed version
+for(int count : counts) {//按照结果简单的第一个空值，发现第一个指定等级的正则返回值，
     sorted_arr.addAll(Collections.nCopies(count, i));
     i++;
     }

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/incoder-6/bucketsort.py
--- 
+++ 
@@ -4,7 +4,8 @@
         counts[x] += 1
 
     sorted_arr = []
-    for i, count in enumerate(arr):
+    for i in range(k):
+        count = counts[i]
         sorted_arr.extend([i] * count)
 
     return sorted_arr
